specification/metadata/data_generator/datageneratorversion.md

```diff
@@ -0,0 +1,17 @@
+# Data Generator Version
```

Comment:
This construct doesn't work. How do they reference the schema as this is not the schema id. The data generator version is insufficient to reference the schema. This change breaks the schema's metadata usefullness.

Reply:
Agreed to merge this PR and open a related issue for this discussion